api/apps/api/src/modules/projects/blm/change-blm-range.command.ts

```diff
@@ -3,8 +3,14 @@ import { Either } from 'fp-ts/Either';
 
 export const invalidRange = Symbol(`invalid range`);
 export const unknownError = Symbol(`unknown error`);
+export const queryFailure = Symbol(
+  `could not query planning unit area for project`,
+);
```

Comment:
How would consumer of this command understand it? (imagine like you as a developer, are using this command via sdk). Does that mean that planning unit could not be found? (then it isn't a query failure, right?)

Reply:
Yep, much clearer!